packages/manager/cypress/support/intercepts/placement-groups.ts

```diff
@@ -175,3 +179,45 @@ export const mockDeletePlacementGroupError = (
     makeErrorResponse(errorMessage, errorCode)
   );
 };
+
+/**
+ * Intercepts PUT request to update Placement Group label and mocks response.
+ *
+ * @param placementGroupId - ID of Placement Group for which to intercept update label request.
+ * @param placementGroupData - Placement Group object with which to mock response.
+ *
+ * @returns Cypress chainable.
+ */
+export const mockUpdatePlacementGroupLabel = (
```

Comment:
```suggestion
export const mockUpdatePlacementGroup = (
```
I think we should rename this (and `mockUpdatePlacementGroupLabelError`) since these utils could be used to mock any PG update request!

Reply:
I agree 👍